SPECS/python3/python3.spec

```diff
@@ -168,8 +168,8 @@ find %{buildroot}%{_libdir} -name '*.o' -delete
 rm %{buildroot}%{_bindir}/2to3
 rm -rf %{buildroot}%{_bindir}/__pycache__
 
-# %check
-# make  %{?_smp_mflags} test
+%check
+%{buildroot}%{_bindir}/python3 -m test
```

Comment:
Does this run all the tests? Or would `-m test.regrtest` or `-m test.pythoninfo` be separate sets of tests?

Reply:
This runs all the tests

Reply:
Some additional documentation on the test package - https://docs.python.org/3/library/test.html#module-test.regrtest

> The [test](https://docs.python.org/3/library/test.html#module-test) package can be run as a script to drive Python’s regression test suite, thanks to the [-m](https://docs.python.org/3/using/cmdline.html#cmdoption-m) option: python -m test. Under the hood, it uses [test.regrtest](https://docs.python.org/3/library/test.html#module-test.regrtest); the call python -m test.regrtest used in previous Python versions still works. Running the script by itself automatically starts running all regression tests in the [test](https://docs.python.org/3/library/test.html#module-test) package. It does this by finding all modules in the package whose name starts with test_, importing them, and executing the function test_main() if present or loading the tests via unittest.TestLoader.loadTestsFromModule if test_main does not exist. The names of tests to execute may also be passed to the script. Specifying a single regression test (python -m test test_spam) will minimize output and only print whether the test passed or failed.

Reply:
Yes, but are we also running the two mentioned? I see that Fedora runs them separately, so I'm curious, if we can test even more.

Reply:
I dove into one of fedora's python3.12.0 builds in koji ([https:/kojipkgs.fedoraproject.org//packages/python3.12/3.12.0/1.fc40/data/logs/x86_64/build.log](https://kojipkgs.fedoraproject.org//packages/python3.12/3.12.0/1.fc40/data/logs/x86_64/build.log)):
- `test.pythoninfo` prints useful debugging information, which would be nice to add. I'll look at adding this in another PR.
- `test.regrtest` runs what appears to be the same tests, though there is a slight difference. In Fedora's build, it is running 483 tests, but in ours we're running 463. I'll see if I can compare the two to figure out the 20 test differences. A random sampling of tests between ours and Fedora's showed that the test scenarios were common